### PR TITLE
Fix bug in `isPrime` and style around

### PR DESCRIPTION
**Pull request by tshemsedinov**
#### Summary

`1` in not a prime number. Before fix `isPrime(1)` return true. After fix `isPrime(1)` return false.

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

And a few style fixes arround.

### CODE REVIEW

File: files/en-us/learn/javascript/asynchronous/introducing/index.md [hunk 1]
```diff
@@ -91,38 +91,36 @@ The program below uses a very inefficient algorithm to generate multiple large p
...
+
+const quota = document.querySelector('#quota');
+const output = document.querySelector('#output');
 
 document.querySelector('#generate').addEventListener('click', () => {
-  const quota = document.querySelector('#quota').value;
-  const primes = generatePrimes(quota);
-  document.querySelector('#output').textContent = `Finished generating ${quota} primes!`;
+  const primes = generatePrimes(quota.value);
+  output.textContent = `Finished generating ${quota} primes!`;
```
Comment (tshemsedinov) [inline files/en-us/learn/javascript/asynchronous/introducing/index.md:119]:
```suggestion
  output.textContent = `Finished generating ${quota.value} primes!`;
```

File: files/en-us/learn/javascript/asynchronous/introducing/index.md [hunk 2]
```diff
@@ -158,38 +156,35 @@ textarea {
...
```
Comment (tshemsedinov) [inline files/en-us/learn/javascript/asynchronous/introducing/index.md:184]:
```suggestion
  output.textContent = `Finished generating ${quota.value} primes!`;
```

File: files/en-us/learn/javascript/asynchronous/introducing/index.md [hunk 3]
```diff
@@ -91,38 +91,36 @@ The program below uses a very inefficient algorithm to generate multiple large p
...
-      if (n % c === 0) {
-          return false;
-       }
-    }
-    return true;
+const MAX_PRIME = 1000000;
+
+const isPrime = (n) => {
+  for (let i = 2; i <= Math.sqrt(n); i++) {
+    if (n % i === 0) return false;
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/asynchronous/introducing/index.md:98]:
```suggestion
    if (n % i === 0) {
      return false;
    }
```

File: files/en-us/learn/javascript/asynchronous/introducing/index.md [hunk 4]
```diff
@@ -158,38 +158,35 @@ textarea {
...
-  function isPrime(n) {
-    for (let c = 2; c <= Math.sqrt(n); ++c) {
-      if (n % c === 0) {
-          return false;
-       }
-    }
-    return true;
+const isPrime = (n) => {
+  for (let i = 2; i <= Math.sqrt(n); i++) {
+    if (n % i === 0) return false;
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/asynchronous/introducing/index.md:165]:
```suggestion
    if (n % i === 0) {
      return false;
    }
```

File: files/en-us/learn/javascript/asynchronous/introducing/index.md [hunk 5]
```diff
@@ -91,38 +91,38 @@ The program below uses a very inefficient algorithm to generate multiple large p
...
 
+const generatePrimes = (quota) => {
+  const primes = [];
   while (primes.length < quota) {
-    const candidate = Math.floor(Math.random() * (maximum + 1));
-    if (isPrime(candidate)) {
-      primes.push(candidate);
-    }
+    const candidate = random(MAX_PRIME);
+    if (isPrime(candidate)) primes.push(candidate);
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/asynchronous/introducing/index.md:111]:
```suggestion
    if (isPrime(candidate)) {
      primes.push(candidate);
    }
```

**Review by Josh-Cena — APPROVED** [review]
This algorithm is super inefficient but I guess it serves its purpose. Thanks for the fix and the refactor!

**Pull request merged**